h/schemas/admin_group.py

```diff
@@ -61,6 +82,15 @@ class CreateAdminGroupSchema(CSRFSchema):
         title=_('Description'),
         description=_('Optional group description'),
         validator=colander.Length(max=GROUP_DESCRIPTION_MAX_LENGTH),
-        widget=TextInputWidget(rows=3),
+        widget=TextAreaWidget(rows=3),
+        missing=None
+    )
+
+    origins = colander.SchemaNode(
```

Comment:
I spent _forever_ trying to get this to work in a way that wasn't hacky. It should really be a `Sequence`, but I couldn't, try as I might, get `Sequence` schemas/nodes to work in our current setup as there is an implicit dependency on certain types of interface (i.e. fields). To explain this in more depth, I'd rather use voices.

Reply:
Follow-up: @robertknight and I used voices and he's going to take a look.